artman/config/proto/config_pb2.py

```diff
@@ -0,0 +1,569 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
I'm not sure about the dynamically loading a `.proto` in the Python proto runtime. I was thinking it would just involve a step of artman that executes `protoc`.